### ISSUE TITLE
Build Fails on Travis

### ISSUE THREAD

**Issue by EliaTraore** [TechnionYP5777/Smartcity-Smarthouse#205]
As you can see [here](https://travis-ci.org/TechnionYP5777/Smartcity-Smarthouse/jobs/244201071),  Tt build is ran for about 15 min and then stopped by travis.
On my local machine the build succeeds - 
```bash
mvn clean package -DskipTests
# lots of line about the build which end with:
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] SmartcitySmarthouse ................................ SUCCESS [  8.134 s]
[INFO] Common ............................................. SUCCESS [  8.331 s]
[INFO] System ............................................. SUCCESS [ 52.154 s]
[INFO] Applications ....................................... SUCCESS [  0.189 s]
[INFO] Sos ................................................ SUCCESS [ 51.867 s]
[INFO] Stove .............................................. SUCCESS [ 12.603 s]
[INFO] Vitals ............................................. SUCCESS [  7.625 s]
[INFO] Shutter ............................................ SUCCESS [  6.548 s]
[INFO] SensorsSimulator ................................... SUCCESS [ 19.596 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:53 min
[INFO] Finished at: 2017-06-18T15:25:32+03:00
[INFO] Final Memory: 52M/346M
[INFO] ------------------------------------------------------------------------

```
anybody got any ideas?

**Comment by RonGatenio** [comment]
[the failed test on travis](https://travis-ci.org/TechnionYP5777/Smartcity-Smarthouse/builds/244228896#L4933)

This test gets stuck in a loop, and there are a lot of exceptions like:
```
java.net.BindException: Address already in use (Bind failed)
	at java.net.PlainDatagramSocketImpl.bind0(Native Method)
	at java.net.AbstractPlainDatagramSocketImpl.bind(AbstractPlainDatagramSocketImpl.java:93)
	at java.net.DatagramSocket.bind(DatagramSocket.java:392)
	at java.net.DatagramSocket.<init>(DatagramSocket.java:242)
	at java.net.DatagramSocket.<init>(DatagramSocket.java:299)
	at java.net.DatagramSocket.<init>(DatagramSocket.java:271)
	at il.ac.technion.cs.smarthouse.system.sensors.SensorsLocalServer.runAddressRequestServer(SensorsLocalServer.java:76)
	at il.ac.technion.cs.smarthouse.system.sensors.SensorsLocalServer.lambda$run$0(SensorsLocalServer.java:37)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by EliaTraore** [comment]
Found the bug! 
Solved at 629d1ef (closing sockets are important)
